scaffolder-templates/github-workflows/advanced-workflow/template.yaml

```diff
@@ -282,10 +282,28 @@ spec:
       name: Generating the Build Code for the Workflow
       action: fetch:template
       if: ${{ parameters.CI == 'tekton_argocd' }}
+      input:
+        url: ../../build
+        values:
+          namespace: ${{ parameters.namespace }}
+          argocdNamespace: ${{ parameters.argocdNamespace }}
+          orgName: ${{ parameters.orgName }}
+          workflowId: ${{ parameters.workflowId }}
+          gitUrl: git@github.com:${{ parameters.orgName }}/${{ parameters.repoName }}.git
+          gitOpsUrl: git@github.com:${{ parameters.orgName }}/${{ parameters.repoName }}-gitops.git
+          quayOrgName: ${{ parameters.quayOrgName }}
+          quayRepoName: ${{ parameters.quayRepoName }}
+          convertToFlat: false
+          gitHost: github.com
+        targetPath: workflow
+    - id: addCIPipeline
+      name: Adding the github pipelinerun
+      action: fetch:template
+      if: ${{ parameters.CI == 'tekton_argocd' }}
       input:
         url: ../build
```

Comment:
is that still OK?

Reply:
ah, this is targeting the github-workflows/build folder, not the build/argocd one, rigth?

Reply:
yes, its referencing the auxiliary build dir that only holds the github pipelines
the gitlab templates are symmetrical